components/api/src/test/java/com/hotels/styx/api/HttpResponseTest.java

```diff
@@ -286,6 +286,16 @@ public void rejectsMultipleContentLength() {
                 .build();
     }
 
+    @Test
+    public void headerValues() {
+        HttpResponse r = response()
+                .addHeader("X-Styx", "14, 16")
+                .addHeader("X-Styx", "15")
+                .build();
+
+        System.out.println("headers: " + r.header("X-Styx"));
```

Comment:
Was this test intended to be committed? The name is bad, there are no assertions and it uses `System.out.println`.